### DISCUSSION TITLE
Projects view

### DISCUSSION THREAD

**Discussion by IgnacioUtrilla** [community/community#3160] [Mobile] [answered]
It could be very useful to see the board project on your phone. This is one of the most attended features in my team. 

Hope to see it soon.

Thank you so much for the hard work 💪

Best regards

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback! Feel free to upvote on this topic related to Projects to help boost awareness of this request! - https://github.com/github/feedback/discussions/3023 
Also appreciate if you can share more details with us in the thread above 👆 - how would you expect to manage projects on mobile? Do you expect to view the full kanban board on mobile? What are the key project features that you think we should cater on mobile?
